deposit/src/main/resources/service-context.xml

```diff
@@ -175,4 +175,55 @@
 	<bean id="pooledHttpClient" class="org.apache.http.impl.client.CloseableHttpClient"
 			factory-bean="pooledHttpClientBuilder" factory-method="build">
 	</bean>
+	
+	<bean id="sparqlQueryService" class="edu.unc.lib.dl.sparql.FusekiSparqlQueryServiceImpl">
+		<property name="fusekiQueryURL" value="${fuseki.baseUri}" />
+	</bean>
+	
+	<!-- Initialize access control dependencies -->
+	<bean id="contentPathFactory" class="edu.unc.lib.dl.fedora.ContentPathFactory"
+			init-method="init">
+		<property name="cacheMaxSize" value="100" />
+		<property name="cacheTimeToLive" value="100" />
+		<property name="queryService" ref="sparqlQueryService" />
```

Comment:
Hmm, if I'm understanding the question correctly you are wondering about `<property ref="sparqlQueryService" />`?  You can see some examples of that pattern in the spring docs, like:
https://docs.spring.io/spring/docs/current/spring-framework-reference/htmlsingle/#beans-factory-instantiation

There shouldn't be any cases of straight up `<ref="anotherExampleBean"/>` though since that's not valid xml